### ISSUE TITLE
Remove include

### ISSUE THREAD

**Issue by netmonitoring** [openssl/openssl#9068]
Branch: master
File: bio.h
Issue: Include file "# include <openssl/e_os2.h>" should be added if OPENSSL_NO_SCTP is defined, but now it is added always because this include added twice, and in first case OPENSSL_NO_SCTP definition ignored.

```
# include <openssl/e_os2.h> // <--

# ifndef OPENSSL_NO_STDIO
#  include <stdio.h>
# endif
# include <stdarg.h>

# include <openssl/crypto.h>
# include <openssl/bioerr.h>

# ifndef OPENSSL_NO_SCTP
#  include <openssl/e_os2.h> // <--
# endif
```

**Comment by levitte** [comment]
There's no reason for the second inclusion to be there.

Ah, that happened in 74a011ebb5e9028ef18982d737a434a8ff926a95, and I think I simply didn't notice the first inclusion when making that change...

**Comment by RameshwarapuSanthosh** [comment]
@netmonitoring,
Can you please close this issue, as it's fixed with: [#9137](https://github.com/openssl/openssl/pull/9137)
